src/state/chat/reducer.js

```diff
@@ -23,6 +22,9 @@ import {
 } from '../action-types';
 import { HAPPYCHAT_CHAT_STATUS_DEFAULT } from '../constants';
 
+const OBVS_UNIX_SEC_DATE = 1700000000;
```

Comment:
I don't understand what's the meaning of this variable name. Would you mind changing it to something more obvious?

Reply:
After reading the thread I see what you're trying to do here: detect whether the client is receiving timestamp in seconds or milliseconds and upscale it if it's seconds, so it's safe to transition the Happychat server to milliseconds.

How about calling the variable `NOVEMBER_14_2023_IN_SECONDS`? Even `FAR_ENOUGH_DATE_IN_SECONDS` might be clear? Also: perhaps is worth using the same variable name across all Happychat codebases, so search is easier.

Also, I think this case merits a comment explaining the rationale for the change and when the code can be deleted.

Reply:
Thanks, absolutely. I realized late at night that I needed to invert the logic--test for low values below an arbitrary "good" seconds date instead of test for a ms value--so I'll blame the rather terrible const name on that.